kerykeion/sign_ingresses/sign_ingress_factory.py

```
@@ -0,0 +1,326 @@
+    return dt
+
+
+def _jd_to_iso(jd: float) -> str:
+    """Convert a Julian Day (UT) to an ISO 8601 UTC string with seconds."""
+    try:
+        dt = julian_to_datetime(jd)
+        return (
+            f"{dt.year:04d}-{dt.month:02d}-{dt.day:02d}"
+            f"T{dt.hour:02d}:{dt.minute:02d}:{dt.second:02d}Z"
+        )
+    except Exception:  # pragma: no cover - defensive
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)